libraries/RC_Channel/RC_Channel.h

```diff
@@ -182,6 +182,8 @@ class RC_Channel {
         Q_ASSIST =            82, // disable, enable and force Q assist
         ZIGZAG_Auto =         83, // zigzag auto switch
         AIRMODE =             84, // enable / disable airmode for copter
+        ACTIVE_PROX_AVOID =   85, // Set active proximity based avoidance
```

Comment:
I'm not sure we need another RC option just for the backaway feature.  I think if the user runs into trouble they can just disable the entire simple avoidance feature.

Reply:
removed